components/CommentsApi.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import UnfoldMoreIcon from '@mui/icons-material/UnfoldMore';
+
+
+export default class CommentsApi extends Component {
+  render() {
+    const comments = this.props.comments;
+    return (
+      <div>
```

Comment:
use Fragment instead of div , as div creates a extra node in DOM and here no css is added in div so Fragment should be used